kafka/consumer_test.go

```diff
@@ -32,9 +33,11 @@ func TestConsumerAPIs(t *testing.T) {
 	}
 
 	c, err = NewConsumer(&ConfigMap{
-		"group.id":           "gotest",
-		"socket.timeout.ms":  10,
-		"session.timeout.ms": 10})
+		"group.id":                 "gotest",
```

Comment:
if you make this a guid you never have to worry about the state of the consumer group when re-running the test.

Reply:
This is a unit-test, it does not interact with any brokers.